example/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "license": "MIT",
   "dependencies": {
-    "stats-webpack-plugin": "^0.4.3",
-    "webpack": "^1.14.0",
-    "webpack-dev-server": "^1.16.2"
+    "stats-webpack-plugin": "*",
```

Comment:
Not loving this, but since we're now using yarn, not as worried about open-ended deps